docs/languages/TypeScript.md

```diff
@@ -0,0 +1,21 @@
+# TypeScript
+There are special use-cases that each language support, therefore this document is for anything related to the TypeScript models.
```

Comment:
There are special use-cases that each language supports; this document pertains to **TypeScript models**.